keras/layers/containers.py

```diff
@@ -316,3 +316,15 @@ def get_config(self):
 
     def count_params(self):
         return sum([layer.count_params() for layer in self.nodes.values()])
+
+    def get_weights(self):
+        weights = []
+        for layer in self.nodes.values():
```

Comment:
This is fine because `nodes` is an OrderedDict. It would be somewhat unsafe for a dict. Maybe that would warrant a code comment.